src/lib/components/messages/typography/index.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from "react"
+import { StyleSheet, Text, TextProperties, TextStyle } from "react-native"
+
+import colors from "../../../../data/colors"
+import fonts from "../../../../data/fonts"
+
+const LargeHeadline = (props: TextProperties) => {
+  const children: string = (props as any).children
```

Comment:
TextProperties has `children` defined (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/bd29808df7f1288d3ed9da358a1b3d6de9f615fd/types/react-native/index.d.ts#L811)
it should be picked up once we update our `@types/react-native` version